Description: Do not build or install sandbox related software, it requires a module not in refpolicy
Author: Russell Coker <russell@coker.com.au>
Last-Update: 2012-09-29


--- a/Makefile
+++ b/Makefile
@@ -1,4 +1,4 @@
-SUBDIRS = sepolicy setfiles semanage load_policy newrole run_init sandbox secon audit2allow sestatus semodule_package semodule semodule_link semodule_expand semodule_deps sepolgen-ifgen setsebool scripts po man gui
+SUBDIRS = sepolicy setfiles semanage load_policy newrole run_init secon audit2allow sestatus semodule_package semodule semodule_link semodule_expand semodule_deps sepolgen-ifgen setsebool scripts po man gui
 
 INOTIFYH = $(shell ls /usr/include/$(DEB_HOST_MULTIARCH)/sys/inotify.h 2>/dev/null)
 
